Duplicati/Server/webroot/ngax/scripts/services/AppUtils.js

```diff
@@ -621,9 +628,11 @@ backupApp.service('AppUtils', function($rootScope, $timeout, $cookies, DialogSer
     };
 
     this.evalFilter = function(path, filters, include) {
-        for(var i = 0; i < filters.length; i++) {
+        for (var i = 0; i < filters.length; i++) {
             var m = path.match(filters[i][1]);
-            if (m && m.length == 1 && m[0].length == path.length)
+            // Regex such as .* might match empty string at the end which is unwanted
+            // Check that the first match covers the full string
```

Comment:
I don't see when there could be several matches, could you please explain ?

Reply:
You can test this on a regex website with JS flavor:
```js
m = "test".match(".*")
m[0] = "test"
m[1] = ""
```

`*` is greedy so it matches as much as possible. After that an empty string is left, which also matches, even though it is counter intuitive.

Reply:
From the comment I had assumed that .* actually meant "something.*" as I could not imagine it was a straight ".*" because such a criterium does not make any sense either for exclusion or inclusion. I had done a test with something actually useful like "a.*" and matches only returns one occurence. 
Could you make the comment more explicit by adding quotes around the RE ? It's nice to add comments but having not so clear comments does not help much in reading the code.